packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/chat.tests.js

```diff
@@ -338,6 +340,17 @@ QUnit.module('Chat', moduleConfig, () => {
             assert.strictEqual($(lastBubble).text(), text);
         });
     });
+
+    QUnit.module('Methods', () => {
+        QUnit.test('The textarea input element must be active after the focus() method is invoked', function(assert) {
+            this.instance.focus();
+
+            const root = document.querySelector('#qunit-fixture');
+            const activeElement = root.shadowRoot ? root.shadowRoot.activeElement : document.activeElement;
+
+            assert.strictEqual(activeElement, this.$input.get(0));
```

Comment:
Don't u also want to check that `dx-state-focused` class is presented here? Because it's even more important (for our styling)

Reply:
dx-state-focused is not set on a <textarea> element (dx-texteditor-input) that is active after the focus() method is called.

dx-state-focused is set on the textarea root element (dx-textarea)

Reply:
https://github.com/DevExpress/DevExtreme/pull/27958#discussion_r1734554909

Reply:
Yeah, sure, `dx-state-focused` should be checked for `dx-textarea`, not input itself